espresso/scripts/startup-with-tee.sh

```diff
@@ -49,6 +49,18 @@ echo "👉 Step 6: Building docker compose..."
 COMPOSE_PROFILES=tee docker compose build
 echo "✅ Docker compose build complete"
 
+# Check for AWS Nitro Enclave support
```

Comment:
Should we have this check earlier, since it's step 5 that usually doesn't succeed?